src/m_misc.c

```diff
@@ -3263,15 +3263,17 @@ void M_LoadDefaults (void)
 
   if (f)
   {
-    I_Printf(VB_INFO, " default file: %s\n", defaultfile);
+    I_Printf(VB_INFO, " default file: %s", defaultfile);
     fclose(f);
   }
   else
   {
-    I_Printf(VB_WARNING, " Warning: Cannot read %s -- using built-in defaults\n",
+    I_Printf(VB_WARNING, " Warning: Cannot read %s -- using built-in defaults",
                          defaultfile);
   }
 
+  I_Printf(VB_INFO, " savegame directory: %s\n", basesavegame);
```

Comment:
Not sure where the best place is to have this reported.

Reply:
I don't think we should be reporting it at all.

Reply:
I don't think we should be reporting it at all.

Reply:
Hm, okay. 🤔